### ISSUE TITLE
Renaming this repository

### ISSUE THREAD

**Issue by BastiaanOlij** [GodotVR/godot-vr-common#22]
I started the code in this repository years ago without really thinking about a name. It was a common repository of VR related code that applies to any platform supported by Godot.

The name doesn't make a lot of sense to people so I am thinking of renaming this repository to `godot-vr-toolkit`. It would be good to get some feedback on this from everyone using this code.

What do you guys think?

**Comment by tomlechner** [comment]
Would xr-toolkit make sense considering the other name changes going on? Is this repo to always be vr only?

**Comment by BastiaanOlij** [comment]
Hey @tomlechner , sorry for the late reply. We did a poll on Twitter and xr-toolkit and xr-tools were suggested a number of times. I've decided to go with godot-xr-tools. 

And yes AR should eventually find its way into this repo as well, a lot of the interaction logic in this repo is applicable for both AR and VR, AR doesn't have much focus yet because not enough people are doing AR stuff with Godot seeing it only really supports ARKit at the moment.
